manifest/net.raidriar796/ResoniteIkCulling/info.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "ResoniteIkCulling",
+	"id": "net.raidriar796.ResoniteIkCulling",
+	"description": "Disables the IK of Users who are behind you or far away. Includes optional IK throttling.",
+	"category": "Optimization",
+	"sourceLocation": "https://github.com/Raidriar796/ResoniteIkCulling",
+	"versions": {
+		"2.4.0": {
```

Comment:
[assembly: AssemblyVersion("0.0.0.0")]

Reply:
Updated both mods, they should have proper assembly info now.